lib/travis/build/addons/jwt.rb

```diff
@@ -0,0 +1,43 @@
+require 'travis/build/addons/base'
+require 'jwt'
+
+module Travis
+  module Build
+    class Addons
+      class Jwt < Base
+        SUPER_USER_SAFE = true
+
+        def before_before_script
+          secrets = nil
+          if config.is_a?(String)
+            secrets = [config]
+          elsif config.is_a?(Array)
+            secrets = config
+          else
+            secrets = config.values
```

Comment:
I am not sure if this should be a desirable default.

As I mentioned previously, I do not think `config` should be a Hash at all, since the keys are discarded.

With that in mind, this portion can be simplified to:

```ruby
secrets = Array(config)
```